src/features/account/AccountTokenList.tsx

```diff
@@ -167,13 +166,15 @@ const AccountTokenList = ({ onLayout }: Props) => {
   const renderItem = useCallback(
     // eslint-disable-next-line react/no-unused-prop-types
     ({ item }: { item: PublicKey }) => {
-      if (GovMints.some((m) => new PublicKey(m).equals(item)))
+      // preferentially show locked HNT
+      if (HNT_MINT.equals(item)) {
```

Comment:
Are you sure this will show GovItem for people who have locked IOT/MOBILE? Seems like it'll only ever show HNT.

Reply:
It won't. They'll have to use the lock button in the token screen.